dist/vendor.sh

```diff
@@ -26,8 +26,10 @@ cargo vendor > "${CARGO_CONFIG}"
 # some vendored files aren't world readable which has lead to issues in container builds when trying out the workflow
 find ./vendor -type f -exec chmod a+r {} \;
 
-git add --force "${FILES_TO_COMMIT[@]}"
-git diff --quiet --staged -- "${FILES_TO_COMMIT[@]}" || {
-  echo Files changed, committing...
-  git commit -m "Update vendored dependencies and rh-manifest.txt" -- "${FILES_TO_COMMIT[@]}"
-}
+if [ "${COMMIT}" = "1" ]; then
```

Comment:
I do not see `COMMIT` defined anywhere.

Reply:
that's correct. it's expected to be passed in the environment to make the script commit. would you prefer having `COMMIT=0` somewhere?

Reply:
Can you give some information about how the environment is going call the script? May be a comment should be ok too.

Reply:
My intent is to just have required information so that when someone want to reuse this script she has the required information.